AndorsTrail/src/com/gpl/rpg/AndorsTrail/controller/CombatController.java

```diff
@@ -406,6 +406,7 @@ private void attackWithCurrentMonster() {
 		} else {
 			combatActionListeners.onMonsterAttackMissed(currentActiveMonster, attack);
 			startMissedEffect(attack, world.model.player.position, this, CALLBACK_MONSTERATTACK);
+			controllers.skillController.applySkillEffectsFromMonsterAttack(attack, currentActiveMonster);
```

Comment:
Although this is correct for now (no skill are used when you are hit by a monster), I think it'd make more sense to move it one line below.

Reply:
This is in the 'else' condition.  When attack misses, check taunt...
This is no different then it was before, except for the order.  Having the taunt applied within the monsterAttacks method required the check for attack.isHit twice.  And it incorrectly logs taunt before miss.

Reply:
This is exactly the issue. 
When attack complete, check skills effect, whatever the result. Let the 'skill-oriented' method take the decisions of what results trigger what skill.

I do think the playerAttacks() method could use the same treatment, as we most likely want any skill effect displayed in the combat log after the hit/crit/miss entry.

Sorry if I'm picky. If you get bored, it's OK to tell me to F-off :laughing: I'll change the code my way anyway :sweat_smile: but I prefer to 'orient' you in the right direction until I merge it, so that your fix remains your fix, your opinion is taken into account, and our visions of the code can align so your (hopefully) next PRs will be merged faster :wink:

Reply:
I'm getting the difference from what I am saying and what you are saying. 
This was/is a very simple solution.  Apply the skill effects from monster attack after the result (hit or miss) of the attack is determined.  This is exactly what I set out to do.  

Fixing the applying of the skill effects from **player** attack is outside the scope of my intention of this change set.  I would recommend making that change as a separate PR.  It's always best to isolate change and narrow scope in case you need/want to rollback.